src/pages/public_registration/PublicRegistration.tsx

```diff
@@ -44,7 +44,9 @@ const PublicRegistration = () => {
 
   return (
     <StyledPageContent>
-      {isLoadingRegistration || isLoadingRegistrationTicketCollection ? (
+      {isLoadingRegistration ||
+      isLoadingRegistrationTicketCollection ||
+      (isRegistrationAdmin && !registrationTicketCollection) ? (
```

Comment:
Dette er for å unngå at siden blinker med innhold før det kommer en ny loading spinner. Problemet er hvis henting av tickets feiler... Da vil denne vise loading spinner for alltid :(

Reply:
Fristet til å se gjennom fingrene for det problemet akkurat nå for å få på plass fiks for dagens problem. Så kan vi løse dette med react-query senere